ci/scripts/collect-deb-package.sh

```diff
@@ -11,15 +11,6 @@ echo "################################################## PACKAGING COLLECT #####
 AUTHOR="Luiz Costa"
 AUTHOR_EMAIL="me@luizgustavo.pro.br"
 
-if ! [ -d ../repo ]; then
-    rm -rf ./repo
-fi
```

Comment:
This is really strange, you check ../repo and you remove ./repo
Was there a typo?